### PR TITLE
JNG-5194 Update versions

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5194" title="JNG-5194" target="_blank">BugJNG-5194</a>  In eclipse the projects reports lifecycle problesms
  </td></table>
  <br />

### CODE REVIEW

**Review by gaborflorian — APPROVED** [review]

File: common-site/category.xml [hunk 1]
```diff
@@ -17,6 +17,13 @@
    -->
    <feature id="org.eclipse.tm.terminal.feature"/>
 
+   <feature id="org.eclipse.gmf.runtime.sdk"/>
+
+<!--
+   <feature id="org.eclipse.gmf"/>
+   <feature id="org.eclipse.gmf.runtime.notation"/>
+   <feature id="org.eclipse.gmf.runtime.thirdparty"/>
+-->
```
Comment (bencegelei) [inline common-site/category.xml:26]: comment?

File: pom.xml [hunk 2]
```diff
@@ -130,7 +136,7 @@
 
         <repository>
             <id>eclipse-m2e</id>
-            <url>https://download.eclipse.org/technology/m2e/releases/2.3.0/</url>
+            <url>https://download.eclipse.org/technology/m2e/releases/latest/</url>
```
Comment (bencegelei) [inline pom.xml:139]: Is this really safe considering versioning anomalies with eclipse related dependencies?:D

Comment (robertcsakany) [reply pom.xml:139]: Usuaally the last is the most prominent for m2e. And if problematic commons will fail ant does not cause regression.

File: common-site/category.xml (hunk 1 above)
Comment (robertcsakany) [reply common-site/category.xml:26]: I'm not sure it will be not mandatory in next release.

**Review by bencegelei — APPROVED** [review]

**Pull request merged**